connexion/resolver.py

```diff
@@ -192,7 +192,8 @@ def post(self):
                     return ...
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args,  class_arguments=None, **kwargs):
+        self.class_arguments = class_arguments or {}
```

Comment:
Could we name this a bit more specific eg. `view_arguments`?
It would also be good to document it here as an argument on top of the documentation you already added.

Reply:
I think class_arguments is more fitting because the arguments for the view function are also called `class_args` and `class_kwargs` that's because these arguments are passed to the class when instantiating it.

Reply:
I will try to add a comment to the init function. I had actually one already with example usage, but pre-commit hook didnt like it.